catalog-info.yaml

```diff
@@ -138,6 +138,15 @@ spec:
         filter_condition: >-
           build.pull_request.id == null || (build.creator.name == 'elasticmachine' && build.pull_request.id != null)
       repository: elastic/elastic-agent
+      schedules:
+        Daily 8_9:
```

Comment:
Do we have to manually update this list  manually whenever a new release branch is created ?

Reply:
Yes, unless this is something that is going to be triggered by the release manager and we don't actually need to do it here, maybe @DaveSys911 can provide some insight.